linode/nb/framework_resource_schema.go

```diff
@@ -104,23 +106,35 @@ var frameworkResourceSchema = schema.Schema{
 			Description: "When this NodeBalancer was last updated.",
 			Computed:    true,
 			CustomType:  timetypes.RFC3339Type{},
+			PlanModifiers: []planmodifier.String{
+				stringplanmodifier.UseStateForUnknown(),
+			},
 		},
 		"tags": schema.SetAttribute{
 			ElementType: types.StringType,
 			Optional:    true,
 			Default:     helper.EmptySetDefault(types.StringType),
 			Computed:    true,
+			PlanModifiers: []planmodifier.Set{
+				linodeplanmodifier.CaseInsensitiveSet(),
+			},
 			Description: "An array of tags applied to this object. Tags are for organizational purposes only.",
 		},
 		"transfer": schema.ListAttribute{
 			Description: "Information about the amount of transfer this NodeBalancer has had so far this month.",
 			Computed:    true,
 			ElementType: TransferObjectType,
+			PlanModifiers: []planmodifier.List{
+				listplanmodifier.UseStateForUnknown(),
+			},
```

Comment:
I'm aware that we can't guarantee the values of these attributes during the plan but this prevents an infinite diff that this resource was running into (see the above comments).

We _might_ be able to implement a custom type diff suppressor to work around this but it would require a good amount of additional boilerplate and would be a bit hacky. I'd definitely appreciate any thoughts on whether this is an acceptable workaround.

Potentially related to https://github.com/hashicorp/terraform-plugin-framework/issues/898